.github/workflows/deploy_docs.yml

```diff
@@ -27,7 +29,11 @@ jobs:
       - name: Build docs
         run: ./gradlew dokkaHtmlMultiModule
 
+      - name: Publish To Maven Local
+        run: ./gradlew publishToMavenLocal
+
       - name: Deploy to Github Pages
+        if: startsWith(github.push.ref, 'refs/tags/')
```

Comment:
FYI in my workflow, I trigger docs on pushes to master, and now on pushes to tags.  If there's no tag for the current commit, the docs are put into a `development` directory.

Reply:
Yeah, I saw that. I'm not doing any sort of versioning of docs right now. So only publishing the latest tag is my current intent. If we need to get fancier about it in the future, we can figure out requirements.